yang-semver/draft-ietf-netmod-yang-semver.txt

```diff
@@ -315,6 +315,29 @@ Internet-Draft                 YANG Semver                 November 2021
       -  "_non_compatible" - the change represents a non-backwards-
          compatible change
 
+   The _COMPAT modifier string is "sticky".  Once a revision of a module
+   has a modifier in the revision label, then all descendants of that
```

Comment:
By "descendants" I get what you are trying to say here, but this could still be confusing, e.g., if you went form 1.2.1_non_compatible to version 3.0.0.  Perhaps say:

, then all descendants of that revision with the same X.Y version prefix will also have a modifier.

Reply:
Good point. I changed it to "with the same X.Y version digits"